redirect-rules/simple/webtask.js

```diff
@@ -0,0 +1,29 @@
+var util = require('util');
+
+module.exports = function (context, req, res) {
+  if (req.method !== 'GET') {
+    res.writeHead(405);
+    return res.end('Method not allowed');
+  }
+
+  var action = util.format('https://%s/continue?state=%s',
+    context.data.auth0_domain,
+    // return whatever state value was passed from the rule
+    context.data.state || '');
+
+  res.writeHead(200, { 'Content-Type': 'text/html '});
+  res.end(
+    '<html>' +
```

Comment:
I would try this approach to render the view: https://github.com/auth0/rules/blob/master/redirect-rules/yubico-mfa.md